src/main/java/com/example/apiportador/presentation/handler/HandlerExceptionAdvice.java

```diff
@@ -0,0 +1,53 @@
+package com.example.apiportador.presentation.handler;
+
+import com.example.apiportador.presentation.handler.exception.ClientDoesNotCorrespondToCreditAnalysisException;
+import com.example.apiportador.presentation.handler.exception.ClientWithIDAlreadyExistsException;
+import com.example.apiportador.presentation.handler.exception.CreditAnalisysNotApproved;
+import com.example.apiportador.presentation.handler.exception.CreditAnalisysNotFoundException;
+import jakarta.validation.ConstraintViolationException;
+import java.net.URI;
+import java.time.LocalDateTime;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ProblemDetail;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class HandlerExceptionAdvice {
+
+    private ProblemDetail builderProblemDetail(String title, HttpStatus status, String detail) {
+        final ProblemDetail problemDetail = ProblemDetail.forStatusAndDetail(status, detail);
+
+        problemDetail.setType(URI.create("https://developer.mozilla.org/pt-BR/docs/Web/HTTP/Status/" + status.value()));
+        problemDetail.setTitle(title);
+        problemDetail.setProperty("timestamp", LocalDateTime.now());
+
+        return problemDetail;
+    }
+
+    @ExceptionHandler(ClientDoesNotCorrespondToCreditAnalysisException.class)
+    public ProblemDetail clientDoesNotCorrespondToCreditAnalysisExceptionHandler(ClientDoesNotCorrespondToCreditAnalysisException cdnctcae) {
+        return builderProblemDetail("Portador não cadastrado", HttpStatus.BAD_REQUEST, cdnctcae.getMessage());
+    }
+
+    @ExceptionHandler(ClientWithIDAlreadyExistsException.class)
+    public ProblemDetail clientWithIDAlreadyExistsExceptionHandler(ClientWithIDAlreadyExistsException cwiaee) {
+        return builderProblemDetail("Portador não cadastrado", HttpStatus.CONFLICT, cwiaee.getMessage());
```

Comment:
Aqui sugiro utilizar 422
Da uma olhada na descrição do 409 [aqui](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409) e [aqui](https://www.httpstatus.com.br/409/).
"current state of the target resource."
"Conflicts are most likely to occur in response to a [PUT](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT) request. For example, you may get a 409 response when uploading a file that is older than the existing one on the server, resulting in a version control conflict."